controllers/model/dashboard_resources.go

```diff
@@ -17,17 +17,6 @@ func GetPluginsConfigMap(cr *grafanav1beta1.Grafana, scheme *runtime.Scheme) *v1
 			Namespace: cr.Namespace,
 		},
 	}
-	controllerutil.SetOwnerReference(cr, config, scheme)
-	return config
-}
-
-func GetDashboardsConfigMap(cr *grafanav1beta1.Grafana, scheme *runtime.Scheme) *v1.ConfigMap {
```

Comment:
This feature isn't used I'm unsure around the logic here, is dashboards supposed to be in a configmap @pb82 ?